pkg/bmt/reference/reference_test.go

```diff
@@ -44,52 +45,56 @@ func sha3hash(t *testing.T, data ...[]byte) []byte {
 func TestRefHasher(t *testing.T) {
 	// the test struct is used to specify the expected BMT hash for
 	// segment counts between from and to and lengths from 1 to datalength
-	for i, x := range []struct {
+	type test struct {
```

Comment:
why revert this change? i inlined them all together so that we have a nice table driven test